VSDiagnostics/VSDiagnostics/VSDiagnostics/VSDiagnosticsResources.resx

```diff
@@ -654,4 +654,14 @@
   <data name="TestMethodWithoutTestAttributeAnalyzerTitle" xml:space="preserve">
     <value>A method might be missing a test attribute.</value>
   </data>
+  <data name="MissingBracesAnalyzerMessage" xml:space="preserve">
+    <value>A{0} {1} statement should be written with braces</value>
+    <comment>{0} - A/An</comment>
+  </data>
```

Comment:
Use the entire word in the argument, that makes it clearer from a caller perspective. so `a` or `an`